src/Main/Results.js

```diff
@@ -217,7 +218,41 @@ class Results extends React.Component {
                       <AboutTab config={config} />
                     )}
                   </div>
+                </div>
+              </div>
+            </div>
+          </div>
 
+          <div className="divider" />
+
+          <div className="row">
+            <div className="col-md-6">
+              <div className="row">
+                <div className="col-md-4">
+                  <div style={{ border: '7px solid #fff', background: 'rgba(0, 0, 0, 0.4)', padding: '8px 14px', fontSize: 40, fontWeight: 700, lineHeight: 1.1 }}>
+                    <Textfit mode="single" max={40}>
+                    How It's<br />
+                    Made
+                    </Textfit>
+                  </div>
+                </div>
+                <div className="col-md-8" style={{ fontSize: 20 }}>
+                  Curious how something is analyzed? Want to better understand your spec? You can find this spec's source <a href={`https://github.com/WoWAnalyzer/WoWAnalyzer/tree/master/${config.path}`}>here</a>.
+                </div>
+              </div>
+            </div>
+            <div className="col-md-6">
+              <div className="row">
+                <div className="col-md-4">
+                  <div style={{ border: '7px solid #fff', background: 'rgba(0, 0, 0, 0.4)', padding: '8px 14px', fontSize: 40, fontWeight: 700, lineHeight: 1.1 }}>
+                    <Textfit mode="single" max={40}>
+                      How To<br />
+                      Change
+                    </Textfit>
+                  </div>
+                </div>
+                <div className="col-md-8" style={{ fontSize: 20 }}>
+                  Spotted a bug? Want a new feature? Suggestion or checklist threshold off? You can investigate the source <a href={`https://github.com/WoWAnalyzer/WoWAnalyzer/tree/master/${config.path}`}>here</a> and a guide on contributing is <a href="https://github.com/WoWAnalyzer/WoWAnalyzer/tree/master/docs#contributing">here</a>.
```

Comment:
If someone spots a bug or has a feature request, the first thing I'd want them to do is come to our Discord and ask about it. I don't think sending them to the source code or the contributing guidelines is the right advice here.